src/sections/reference.html

```diff
@@ -0,0 +1,39 @@
+<div
+  class="bg-cover bg-center h-full bg-[url('../../../img/Rectangle-3.webp')]"
+>
+  <div class="flex justify-end">
+    <ul
+      class="font-outfit text-right pb-48 px-10 text-xl pt-20 md:text-2xl md:px-32 lg:px-40"
```

Comment:
Instead of pt-20, you can use py-20 and delete pb-48, and everywhere you used px replace it with pr, since we only need padding-right not on both sides.